include/surface_terrains.h

```diff
@@ -56,7 +56,7 @@ enum SurfaceTypes {
     SURFACE_LOOK_UP_WARP,               // 0x002F // Look up and warp (Wing cap entrance)
     SURFACE_HARD,                       // 0x0030 // Hard floor (Always has fall damage)
     SURFACE_0031,                       // 0x0031 // Unused
-    SURFACE_WARP,                       // 0x0032 // Surface warp
+    SURFACE_WARP,                       // 0x0032 // Surface warp. Warps to ID of force parameter's second byte if set, otherwise defaults to ID 0xF3.
```

Comment:
What about vertical wind floors?

Reply:
vertical wind floors have the same functionality as death planes in regards to warping

Reply:
Exactly, but that's not mentioned anywhere here

Reply:
fixed